### DISCUSSION TITLE
Can't find "Send At" biutton

### DISCUSSION THREAD

**Discussion by mainegeezxer** [Extended-Thunder/send-later#612] [General]
The "send later" window has a couple of pop-ups, for example

Clicking “Put in Outbox” puts the message in your Outbox immediately. It DOES NOT schedule the message to be sent at a specific time. For that, click the “Send at” button in the green box after entering a valid send time. Are you sure you want to put the message in your Outbox now?

and 

Clicking “Put in Outbox” puts the message in your Outbox immediately. It DOES NOT schedule the message to be sent at a specific time. For that, click the “Send at” button in the green box after entering a valid send time. Are you sure you want to put the message in your Outbox now?

I see no green  box, I see no "Send At" button

What am I missing?

**Comment by jikamens** [comment]
Oh, sorry, I didn't realize that old language was still the tooltip. I'll remove it, since you're right, the box is gone.

The send at button is the one the arrow is pointing at in this screen shot:

![image](https://github.com/Extended-Thunder/send-later/assets/1598067/7fba7c16-3675-4576-ab71-48d2fef9bbf7)

**Reply by mainegeezxer** [reply]
Thank you!

**Comment by mainegeezxer** [comment]
Another question: How does one turn off end-to-end encryption?  If I  go to Account Settings -> End-to-End Encryption and look under Advanced Settings, there's a checkbox, which is checked,  for "Store draft messages in encrypted format" but it is faded out and can't be changed. 

Do I have to delete all the stored certificates? I don't consciously use encryption; I'm not sure where all the stored certificates came from, except maybe years ago I was messing about and have forgotten about it. Will I break anything by deleting them?

**Comment by jikamens** [comment]
Please see https://github.com/Extended-Thunder/send-later/discussions/608#discussioncomment-6858001
